lib/isolator/ignorer.rb

```diff
@@ -18,7 +18,7 @@ class << self
       def prepare(path:, regex_string: "^.*(#ignores#):.*$")
         return unless File.exist?(path)
 
-        ignores = YAML.load_file(path)
+        ignores = YAML.load_file(path, aliases: true)
```

Comment:
We need to support older Rubies. And it's a bit tricky) Take a look at the example here: https://github.com/palkan/anyway_config/blob/8380a73ee09fb3e78baafd8e7b45148bc8a99a34/lib/anyway/loaders/yaml.rb#L47